src/main.rs

```diff
@@ -87,24 +90,35 @@ impl Deref for Config {
 }
 
 pub trait PackageExt {
-    fn is_third_party(&self, policy: &SortedMap<PackageName, PolicyEntry>) -> bool;
+    fn is_considered_third_party(&self, policy: &Policy) -> bool;
+    fn is_third_party(&self) -> bool;
```

Comment:
Maybe simply renaming to `is_crates_io` or `is_from_crates_io` would both be more specific and would avoid accidental usage (since it's different terminology than "third-party")?